cmake/modules/AddCephTest.cmake

```diff
@@ -5,9 +5,10 @@ function(add_ceph_test test_name test_path)
   add_test(NAME ${test_name} COMMAND ${test_path} ${ARGN})
   if(TARGET ${test_name})
     add_dependencies(tests ${test_name})
+    set_property(TARGET ${test_name}
+      PROPERTY EXCLUDE_FROM_ALL TRUE)
```

Comment:
Personally I'm for it :+1:; just want to notice it actually can be surprising to developers. Notifying folks first would be a good idea.

Reply:
will send a mail to ceph-devel once this change is merged.